### PR TITLE
[Zone] Felwood

### PR DESCRIPTION
**Pull request by Knindzagxg**
Felwood is now fully updated, since every creature in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Felwood and run / test around.

### CODE REVIEW

**Comment by Kitzunu** [comment]
2020-05-10 14:49:25 SmartAIMgr: EntryOrGuid 7106 using event(1) has an invalid action type (0), skipped.

File: data/sql/updates/pending_db_world/rev_1589121823177850900.sql [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+(7110, 0, 2, 0, 0, 0, 100, 0, 2600, 3100, 19800, 20100, 11, 14897, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Shadowstalker - In Combat - Cast \'14897\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7106;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7106);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7106, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 13578, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Rogue - On Aggro - Cast \'13578\''),
+(7106, 0, 1, 0, 2, 0, 100, 1, 15, 30, 0, 0, 0, 13579, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Rogue - Between 15-30% Health - No Action Type (No Repeat)');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1589121823177850900.sql:196]: 1. This makes no sense and is causing errors
2. A complete version already exists in the DB.

What I suggest:
Check the other SAI's if they already exist in DB and remove this one

**Comment by Knindzagxg** [comment]
It's Gauge spell that making problems cause it will put creature out of combat. Same for Backstep cause need to behind.
Ill sort once i'm done with work.

File: data/sql/updates/pending_db_world/rev_1589121823177850900.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1589121823177850900.sql]: Sorted.

Sadly, we can't use Gauge cause it make problems.

**Comment by Geekena** [comment]
damage and spells are ok
no problems aswell

**Pull request merged**